### PR TITLE
Feature/get custom properites

### PR DESCRIPTION
**Pull request by bishopbm1**
Added new action to query node custom properties for the specified property and return the result. Added tests for new method.

### CODE REVIEW

File: actions/get_node_custom_property.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        FROM Orion.NodesCustomProperties
+        WHERE NodeID={0}""".format(orion_node.npm_id, custom_property)
+        data = self.query(swql)
+
+        if 'results' not in data:
+            msg = "No results from Orion: {}".format(data)
+            self.logger.info(msg)
+            raise Exception(msg)
+
+        if len(data['results']) == 1:
```
Comment (nmaludy) [inline actions/get_node_custom_property.py:43]: why not just return the array here?

File: actions/get_node_custom_property.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+runner_type: "python-script"
+
+parameters:
+  node:
+    type: "string"
+    description: "The node to poll in Orion."
+    required: true
+  custom_property:
+    type: "string"
+    description: "The custom property to update"
```
Comment (nmaludy) [inline actions/get_node_custom_property.yaml]: are we updating the custom property here?

Comment (nmaludy) [reply actions/get_node_custom_property.yaml]: should we have a custom property set also?

**Review by nmaludy — CHANGES_REQUESTED** [review]

File: actions/get_node_custom_property.yaml (hunk 2 above)
Comment (bishopbm1) [reply actions/get_node_custom_property.yaml]: Fixed the wording there. There is already a custom set method.

**Review by nmaludy — APPROVED** [review]

**Pull request merged**